CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing
+
+Contributions are welcome! Please see the [development section](#development) below for more information. For new bpftrace tools, please add them to the new [user-tools repository](https://github.com/bpftrace/user-tools/blob/master/CONTRIBUTING.md). The tools that exist in this repository are a small collection curated by the bpftrace maintainers.
+
+* Bug reports and feature requests: [Issue Tracker](https://github.com/bpftrace/bpftrace/issues)
+
+* Development IRC: #bpftrace at irc.oftc.net
+
+* [Good first issues](https://github.com/bpftrace/bpftrace/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
+
+## Development
+
+* [Coding Guidelines](docs/coding_guidelines.md)
+* [Development Guide](docs/developers.md)
+* [Development Roadmap](https://docs.google.com/document/d/17729Rlyo1xzlJObzHpFLDzeCVgvwRh0ktAmMEJLK-EU/edit)
+* [Fuzzing](docs/fuzzing.md)
+* [Nix](docs/nix.md)
+* [Release Process](docs/release_process.md)
+* [Tests](tests/README.md)
+
+## Developer's Certificate of Origin
+
+Any contributions to bpftrace must only contain code that can legally be contributed to bpftrace, and which the bpftrace project can distribute under its license.
```

Comment:
That's not right.
Just copy paste the two paragraphs from:
https://docs.kernel.org/process/submitting-patches.html#sign-your-work-the-developer-s-certificate-of-origin

but skip 'Any further SoBs' part, since it applies to manual git style and doesn't quite work with github model.

Reply:
I took parts of those first two paragraphs, removing the kernel and patch specific bits.